apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -549,10 +549,9 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
     }
 
     public submit() {
-
-        this.createAccountIsSuccess$.subscribe(res => {
-            if (res) {
-                // This will reset the active group after submit 
+        this.store.pipe(select(s => s.groupwithaccounts.createAccountIsSuccess), takeUntil(this.destroyed$)).subscribe(response => {
```

Comment:
Consider removing the `@Input` property `createAccountIsSuccess$` from the component and parent component `accounts-operation.component`

Reply:
Sir this is already assigned there